dev-tools/iota-indexer-monitoring/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+      - prometheus_data:/prometheus
+    ports:
+      - "9090:9090"
+    restart: always
+    networks:
+      - indexer-metrics-net
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    environment:
```

Comment:
Just a hint, if you want to point to a specific dashboard per default, then you can use the following ENV var:
```
      - GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH=/etc/grafana/provisioning/dashboards/node_dashboard.json
```

Reply:
Added by https://github.com/iotaledger/iota/pull/6378/commits/2380311700a8cfff614c9148e917c2cf660fd802